adapters/spi/jboss-adapter-core/pom.xml

```diff
@@ -30,6 +30,10 @@
     <name>Common JBoss/Wildfly Core Classes</name>
     <description/>
 
+    <properties>
+        <maven.compiler.release>11</maven.compiler.release>
```

Comment:
Same here, should we log an issue to make this compile with a more modern version instead?

Reply:
@jonkoops Still not sure about the EAP SAML adapter; needs to be verified.

Reply:
Just FYI the reason why this one needs to be down-leveled is because the SAML adapter makes use of [`java.security.acl.Group`](https://docs.oracle.com/en%2Fjava%2Fjavase%2F11%2Fdocs%2Fapi%2F%2F/java.base/java/security/acl/Group.html), which was removed in JDK 14.

Reply:
Just a note: This needs to stay on JDK 11 as this is lowest version supported by WF/EAP8.

Reply:
I think we can resolve this thread, as we agree the minimum version will remain 11 until EAP upgrades to a newer version.